### ISSUE TITLE
Unclear warning messages

### ISSUE THREAD

**Issue by alex9138** [scolladon/sfdx-git-delta#37]
Hello @scolladon ,

Got this return on a diff : 

```
 sgd -t HEAD -f 66dda8e00810af448bc42526894a50845f66528a -r . -o output
{
  "error": null,
  "output": "output",
  "success": true,
  "warnings": [
    "metadataContent[subType].forEach is not a function",
    "metadataContent[subType].forEach is not a function"
  ]
}
```

For repro you can go on develop branch

**Comment by scolladon** [comment]
@alex9138 there is a PR to improve information displayed #36 

In the meantime, could you :

1. Make sure your repository metadata are correct ? (It compiles if you try to deploy it)
2. Make sure the delta produced by the CLI is correct ? (or does it misses elements). If some elements are missing it can be a clue to isolate and more accurately reproduce the issue
3. If you reach this point, there is a probably an issue with the tool itself, so if you manage to isolate and provide a reproducible repository state it will greatly help us

Thanks in advance !

**Comment by alex9138** [comment]
1) Work in progress, some errors in the full deploy (3300 metas), but full deploy was working before so we will be able to repair it soon

**Comment by scolladon** [comment]
Resolution is merged (#36) and will be included in the next release
